shared/featureconfig/flags.go

```diff
@@ -5,6 +5,11 @@ import (
 )
 
 var (
+	// CacheTreeHashFlag determines whether to cache tree hashes for ssz.
+	CacheTreeHashFlag = cli.BoolFlag{
+		Name:  "cache-tree-hash",
```

Comment:
Please use the format `--enable-cache-tree-hash`

Reply:
thanks